### PR TITLE
Register branch update

### PR DESCRIPTION
**Pull request by pepijnkok**
Updated register and index page, added mongoose, register scheme and small css changes

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -10,16 +10,32 @@ const upload = multer({
...
+})
+
+db.on('connected', () => { 
+  console.log('Mongoose connected')
+})
 
 // Mongodb
 const MongoClient = require('mongodb').MongoClient;
 const uri = process.env.DB_URI;
 const client = new MongoClient(uri,{ useUnifiedTopology: true });
```
Comment (daoneandonly) [inline app.js]: If we were to use mongoose over MongoDB, we should remove anything that makes a connection through MongoDB. Don't worry about dependencies (in `package.json`), I'll make sure that works. For now just make sure variables and references to MongoDB are removed.

File: app.js [hunk 2]
```diff
@@ -10,16 +10,32 @@ const upload = multer({
 const dotenv = require('dotenv');
 dotenv.config();
 
-
 app.set('view engine', 'ejs');
 app.use(express.static('static'));
 
+// Mongoose
+const mongo = require('mongodb')
```
Comment (daoneandonly) [inline app.js:17]: If we were to use mongoose over MongoDB, we should remove anything that makes a connection through MongoDB. Don't worry about dependencies (in `package.json`), I'll make sure that works. For now just make sure variables and references to MongoDB are removed.

**Review by daoneandonly — CHANGES_REQUESTED** [review]
Very clean code! 👏 Let's remove everything MongoDB related if gonna use mongoose to make a connection.

File: app.js (hunk 2 above)
Comment (pepijnkok) [reply app.js:17]: I agree, but if we delete MongoDB the develop will break, so we first need to change all the other code that uses the MongoDB client and switch it to Mongoose

**Pull request merged**